src/pages/security/ddos-denial-of-service-attacks/index.md

```diff
@@ -2,14 +2,13 @@
 title: Ddos Denial of Service Attacks
 ---
 ## Ddos Denial of Service Attacks
-
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/security/ddos-denial-of-service-attacks/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
-
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
-
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
-
-#### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
-
-
+ 
+ 
+      Distributed DoS attack. A distributed denial-of-service (DDoS) attack occurs when multiple systems flood the bandwidth or resources of a targeted system, usually one or more web servers. Such an attack is often the result of multiple compromised systems (for example, a botnet) flooding the targeted system with traffic. Also it dosen't need to be a multiple system in the case of zip bomb.
+      
+ # What is Zip bomb?
```

Comment:
While you've shown how a zip bomb can also help to deny service, it is not distributed, and therefore not DDOS.